release/rllib_tests/learning_tests/yaml_files/appo/appo-pongnoframeskip-v5.yaml

```diff
@@ -34,6 +34,7 @@ appo-pongnoframeskip-v5:
             dim: 42
             conv_filters: [[16, 4, 2], [32, 4, 2], [64, 4, 2], [128, 4, 2]]
             conv_activation: relu
+            post_fcnet_hiddens: [256]
```

Comment:
Now that we don't have an automatic and implicit (which is bad) last dense layer in the Conv2d stack, we need to make sure that pi- and vf-heads have their dense layers.

Yes, this is a slight change to the current (new stack) architecture, from:
```
encoder: conv2d conv2d conv2d dense(256) # <- this 256 is non-intuitively set under the hood by RLlib because fcnet_hiddens - which has NOTHING to do with this setup - by default is [256, 256] and we take the -1th item :(
pi: dense([num_actions])
vf: dense(1)
```

To:
```
encoder: conv2d conv2d conv2d
pi: dense(256) dense([num_actions])
vf: dense(256) dense(1)
```

But at least, all settings are explicit right now.